components/x-podcast-launchers/src/PodcastLaunchers.scss

```diff
@@ -0,0 +1,52 @@
+:global {
+	@import "~@financial-times/x-follow-button/dist/FollowButton";
+}
+
+.container {
+	font-family: MetricWeb,sans-serif;
+	width: 100%;
+}
+
+.heading {
+	font-weight: 600;
+	font-size: 18px;
+	line-height: 20px;
+	margin: 0 0 12px;
+}
+
+.podcast-app-links__wrapper {
+	list-style: none;
+	padding: 8px 0 0;
+	margin: 0;
+}
+
+.podcast-app-link {
+	width: 100%;
+	margin-bottom: 8px;
+}
+
+.rss-url__wrapper {
+	margin-top: 0px;
+	margin-bottom: 24px;
+}
+
+.rss-url__input {
+	max-width: none;
+}
+
+.rss-url__copy-button {
+	padding-left: 0px;
+}
+
+.rss-url__copy-span {
```

Comment:
Question: Is there a reason to use BEM style class names rather than JS compatible identifiers, e.g. `.rssUrlCopyButton`?